stable/portus/templates/minio-create-bucket-job.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.minio.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}-create-bucket"
+  labels:
+    app: "{{ template "portus.name" . }}"
+    chart: {{ template "portus.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: minio
+  annotations:
+    "helm.sh/hook": post-install
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+        app: "{{ template "portus.name" . }}"
+        component: minio
```

Comment:
Add `release` label to selector.

Reply:
Thanks for the reminder.
I will get that fixed.